site/content/en/guides/producer/setters/_index.md

```diff
@@ -181,6 +192,20 @@ gcloud.compute.region
 gcloud.compute.zone
 ```
 
+#### Inherit Setter Values from Parent Package
+
+##### Notice: This is an experimental feature and is subjected to changes soon
```

Comment:
So this isn't really a guide, this is more like a reference documentation. This is part of a bigger issue about the guides, in particular the producer guides for setter and substitutions. Can we take a pass at updating those guides and make sure they look like a guide, i.e. one end-to-end workflow that accomplishes something. We might need to split the content up into more guides.

Reply:
+1. Guides are tutorials to accomplish a concrete goal. They're different from references.